### PR TITLE
Removing Nose-Progressive Dependency

### PR DESCRIPTION
**Pull request by archetypalsxe**

### CODE REVIEW

File: records_mover/records/mover.py [hunk 1]
```diff
@@ -115,7 +115,8 @@ def move(records_source: RecordsSource,
                                         override_records_format=records_source.records_format,
                                         processing_instructions=processing_instructions)
     elif isinstance(records_source, SupportsToFileobjsSource):
-        target_records_format: BaseRecordsFormat = getattr(records_target, "records_format", None)
+        # Incompatible types in assignment (expression has type "Optional[Any]", variable has type "BaseRecordsFormat")
```
Comment (bluelabsbutler) [inline records_mover/records/mover.py:118]: E501 line too long (119 > 100 characters)

File: records_mover/airflow/hooks/records_hook.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
     from airflow.hooks import BaseHook
 except ImportError:
     # Required for Airflow 2.0
-    from airflow.hooks.base_hook import BaseHook
+    from airflow.hooks.base_hook import BaseHook # type: ignore
```
Comment (bluelabsbutler) [inline records_mover/airflow/hooks/records_hook.py:17]: E261 at least two spaces before inline comment

File: records_mover/airflow/hooks/sqlalchemy_db_hook.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
     from airflow.hooks import BaseHook
 except ImportError:
     # Required for Airflow 2.0
-    from airflow.hooks.base_hook import BaseHook
+    from airflow.hooks.base_hook import BaseHook # type: ignore
```
Comment (bluelabsbutler) [inline records_mover/airflow/hooks/sqlalchemy_db_hook.py:9]: E261 at least two spaces before inline comment

File: records_mover/records/mover.py [hunk 4]
```diff
@@ -115,7 +115,9 @@ def move(records_source: RecordsSource,
                                         override_records_format=records_source.records_format,
                                         processing_instructions=processing_instructions)
     elif isinstance(records_source, SupportsToFileobjsSource):
-        target_records_format: BaseRecordsFormat = getattr(records_target, "records_format", None)
+        # Incompatible types in assignment (expression has type "Optional[Any]",
+        #     variable has type "BaseRecordsFormat")
+        target_records_format: BaseRecordsFormat = getattr(records_target, "records_format", None) # type: ignore
```
Comment (bluelabsbutler) [inline records_mover/records/mover.py:120]: E261 at least two spaces before inline comment

Comment (bluelabsbutler) [inline records_mover/records/mover.py:120]: E501 line too long (113 > 100 characters)

File: records_mover/records/targets/google_sheets.py [hunk 5]
```diff
@@ -103,7 +103,8 @@ def _get_service(self) -> SheetsService:
 
     def as_json_serializable(self, cell: Any) -> Any:
         if isinstance(cell, np.generic):
-            native = np.asscalar(cell)
+            # MyPy complains that this method does not exist
+            native = np.asscalar(cell) # type: ignore
```
Comment (bluelabsbutler) [inline records_mover/records/targets/google_sheets.py:107]: E261 at least two spaces before inline comment

**Comment by ryantimjohn** [comment]
I'm going to take this on and try and get the CircleCI tests above to pass!

**Comment by ryantimjohn** [comment]
Alright @mitchellj, I feel ready for this to be reviewed. Happy to talk through and explain any decisions made.

**Review by mitchellj — APPROVED** [review]

**Pull request merged**